package.json

```diff
@@ -43,6 +43,7 @@
     "eslint-plugin-promise": "^6.1.1",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "node-fetch": "^3.3.1",
```

Comment:
Why install node-fetch?

Fetch is available by default now. https://blog.logrocket.com/fetch-api-node-js/

Reply:
Oops, nice.